### DISCUSSION TITLE
Cookie popup is not displayed if browser running using selenium webdriver

### DISCUSSION THREAD

**Discussion by DaryaGorbachyova** [orestbida/cookieconsent#474] [Q&A] [answered]
Hi. I'm trying to run automated tests for cookie plugin but it is not diplayed on Chrome. I run Chrome using this properties:
 "options": {
        "intl.accept_languages": "en",
        "safebrowsing.enabled": "false",
        "profile.default_content_settings.popups": "0",
        "profile.cookie_controls_mode": 0,
        "profile.block_third_party_cookies": false,
        "disable-popup-blocking": "false",
        "download.prompt_for_download": "false",
      },
      "startArguments": [
        "--no-sandbox" 
      ],
      "pageLoadStrategy": "Normal" 

What argument should I use for popup diplaying?

**Comment by orestbida** [comment, marked as answer]
That's the default expected behavior. You can set `hide_from_bots: false` to make it work in selenium webdriver.
